{{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/common/main.tf

```diff
@@ -19,10 +19,18 @@ resource "databricks_secret" "cd_credentials_token" {
   scope        = databricks_secret_scope.cd_credentials.id
 }
 
+resource "databricks_secret" "cd_github_server" {
+  key = "github_server"
+  // regex expression for parsing github server url out of github repo url
+  // example: http(s)://mygitserver.anything/org/repo -> http(s)://mygitserver.anything
+  string_value = regex(".*\/\/[^\/]+", var.github_repo_url)
```

Comment:
This regex scares me a bit, can we confirm this works as expected via some tests? Wonder if it would be simpler to ask them to provide it

Reply:
Cool, will add another variable for github_server